scripts/install/functions.sh

```diff
@@ -193,13 +185,18 @@ function install_from_sources {
 
     npm set @spartacus:registry http://localhost:4873/
 
-    clone_repo
```

Comment:
Is there a need to do a `git pull` to get up to date code as when cloning the repo?

Reply:
I thought about adding a comment on the README of having your repo up to date. Thoughts?

Reply:
Ok to me.  We can add a git pull here if it seems useful